cabal-install/src/Distribution/Client/Init/Format.hs

```diff
@@ -328,7 +329,9 @@ mkPkgDescription opts pkgDesc =
       False
       opts
 
-    , field  "license" pretty (_pkgLicense pkgDesc)
+    , (if _pkgCabalVersion pkgDesc < CabalSpecV2_2
+        then field "license" pretty (licenseFromSPDX $ _pkgLicense pkgDesc)
```

Comment:
There is `prettyVersioned`

Reply:
And https://hackage.haskell.org/package/Cabal-3.6.2.0/docs/Distribution-FieldGrammar-Newtypes.html#t:SpecLicense

Reply:
I'll take a look into it.